src/Widgets/SaveDialog.vala

```diff
@@ -73,6 +73,16 @@ public class Screenshot.SaveDialog : Gtk.Dialog {
         preview_box_context.add_class (Granite.STYLE_CLASS_CARD);
         preview_box_context.add_class (Granite.STYLE_CLASS_CHECKERBOARD);
 
+        var preview_event_box = new Gtk.EventBox ();
+        preview_event_box.add (preview_box);
```

Comment:
It looks like eventbox clips box shadows, so this probably needs to go inside `preview_box` instead of the other way around

Reply:
I have just done that and it does preserve the box shadow effect:

![Screenshot from 2020-07-13 18-49-37](https://user-images.githubusercontent.com/221446/87357232-eee4c800-c539-11ea-9bd3-59db3da437f9.png)